news.html

```diff
@@ -8,13 +8,21 @@
 
     <div class="container community-wrapper">
 
-        <div class="col-sm-8 wrapper">
+        <div class="col-sm-12 wrapper">
```

Comment:
I disagree. It makes that table float and look like there is something missing on the page.

The way bootstrap works is you use CSS classes to build a grid layout. Giving something a class of `col-sm-8` implies you'll put a column with `col-sm-4` or similar before or after.

If you look at `index.html` you can see how the first column is a 4/12 and then the second is 8/12. This was mistakenly set at 8 and should be a 12.